CHANGELOG.md

```diff
@@ -1,2136 +1,2132 @@
+- Multiline string in use expression. [\#1055](https://github.com/fsprojects/fantomas/issues/1055)
+- Issue #246 has returned in v4.0.0. [\#1051](https://github.com/fsprojects/fantomas/issues/1051)
+- Indentation after multiple hash directives is off. [\#1026](https://github.com/fsprojects/fantomas/issues/1026)
+- if/then/else indented too far. [\#1054](https://github.com/fsprojects/fantomas/issues/1054)
+- Single AST node should contain trivia. [\#1031](https://github.com/fsprojects/fantomas/issues/1031)
+- Formatter adds extra newlines between type and any subsequent val in .fsi files. [\#1029](https://github.com/fsprojects/fantomas/issues/1029)
+- Comments in if/then/else statements are sometimes deleted. [\#1019](https://github.com/fsprojects/fantomas/issues/1019)
+- Moves type name around when writing constrained type defintions. [\#1018](https://github.com/fsprojects/fantomas/issues/1018)
+- Line break before bracket on long method call followed by member access causes semantic change. [\#994](https://github.com/fsprojects/fantomas/issues/994)
+- Long union case should be split over multiple lines. [\#972](https://github.com/fsprojects/fantomas/issues/972)
+- FSI formatting does the wrong thing with comments on single-case DU. [\#965](https://github.com/fsprojects/fantomas/issues/965)
+- Invalid unit test ``should break lines on multiline if conditions``. [\#863](https://github.com/fsprojects/fantomas/issues/863)
+- Abstract member declarations don't follow page width. [\#435](https://github.com/fsprojects/fantomas/issues/435)
 
 ## [4.0.0] - 2020-08-27
 
 ### Changed
-* Add initial support of String Interpolation. [#998](https://github.com/fsprojects/fantomas/issues/998)
-* Extract FakeHelpers and EditorConfig to Fantomas.Extras project [#986](https://github.com/fsprojects/fantomas/issues/986)
-* Update FCS to 37. [#996](https://github.com/fsprojects/fantomas/pull/996)
-* Update FCS to 36. [#899](https://github.com/fsprojects/fantomas/pull/899), [#961](https://github.com/fsprojects/fantomas/pull/961)
-* Replaced json configuration with .editorconfig. [#650](https://github.com/fsprojects/fantomas/issues/650)
-* Sunset setting ReorderOpenDeclaration. [#645](https://github.com/fsprojects/fantomas/issues/645)
-* Sunset setting KeepNewlineAfter. [#737](https://github.com/fsprojects/fantomas/issues/737)
-* Renamed setting IndentSpaceNum to IndentSize. [#940](https://github.com/fsprojects/fantomas/pull/940)
-* Renamed setting PageWidth to MaxLineLength. [#940](https://github.com/fsprojects/fantomas/pull/940)
-* Removed all style configuration options from CLI tool. [#704](https://github.com/fsprojects/fantomas/issues/704)
-* Added [Benchmarks](https://fsprojects.github.io/fantomas/). [#867](https://github.com/fsprojects/fantomas/issues/867)
-* Split up setting SpaceBeforeArgument to multiple settings. [#649](https://github.com/fsprojects/fantomas/pull/649)
-  - SpaceBeforeParameter
```

Comment:
The new changelog parser doesn't seem to like the nested lists here - it complains about there not being a dash at the start of the line - not sure what syntax needs to be used to keep that happy, whilst still rendering the child lists nicely in the Github formatted markdown view